packages/viewer/src/scenes/queries.ts

```diff
@@ -148,4 +148,16 @@ export class SceneItemQueryExecutor {
       expression
     );
   }
+
+  public all(): SceneItemOperationsExecutor {
+    const allExpresion: QueryExpression = {
+      type: 'all',
+    };
+
+    return new SceneItemOperationsExecutor(
+      this.sceneViewId,
+      this.commands,
+      allExpresion
+    );
+  }
```

Comment:
We shouldn't need this method. There's already an `RootQuery.all` that can be accessed via: `scene.items().where(q => q.all())`. With this, there'd be two places where query expressions can be defined. Should keep it to a single place so things are consistent.

Reply:
Yeah, I had considered that as an alternative option. If that's the way we wanna represent that, sounds good to me! 👍